src/Grok.Net/Grok.cs

```diff
@@ -33,6 +40,32 @@ public Grok(string grokPattern, Stream customPatterns)
         {
             LoadCustomPatterns(customPatterns);
         }
+        
+        public Grok(string grokPattern, IEnumerable<(string, string)> customPatterns)
```

Comment:
> Could you please explain why not use IDictionary<string, string>?

Nothing special. I try to prevent Dictionaries because there are not made for Concurrency (except `ConcurrentDictionary`) and I like using LINQ in its raw form. But since it is internally converted into a dictionary, there is no reason to keep that. It should be fixed in the latest commit.